### DISCUSSION TITLE
windows下编译的网页打开会白屏

### DISCUSSION THREAD

**Discussion by LoveBeforT** [AlistGo/alist#186] [Q&A] [answered]
### Alist Version / Alist 版本

最新代码

### Describe the bug / 问题描述

go：1.17.3

mac下编译：一切正常
Ubuntu2004下编译：一切正常
win10下编译：网页打开会白屏，chrome F12调试里面能看到js文件正常接收了，但是不显示
如果用releases里面的最新windows exe能正常显示网页，自己编译的不行

**Comment by xhofe** [comment]
提供一下编译后的文件

**Comment by LoveBeforT** [comment]
[public.zip](https://github.com/Xhofe/alist/files/7646625/public.zip)

**Comment by xhofe** [comment]
> [public.zip](https://github.com/Xhofe/alist/files/7646625/public.zip)

并没有问题 可以打开

**Comment by LoveBeforT** [comment]
![image](https://user-images.githubusercontent.com/13610008/144546805-7692421d-894f-4d5f-ae8e-e62c9e0d19b5.png)
![image](https://user-images.githubusercontent.com/13610008/144546824-239fbdec-7702-4f92-b17a-4af2077f3c9f.png)
但是我这里打不开

**Comment by xhofe** [comment, marked as answer]
https://stackoverflow.com/questions/54835510/getting-mime-type-text-plain-error-in-golang-while-serving-css
https://github.com/golang/go/issues/32350
> I am using a windows machine (windows 10 & windows server 2019) and I had the same issue on javascript files, I went into the registry \HKEY_CLASSES_ROOT\.js > "Content Type" and i changed it from "text/plain" to "application/javascript" and i restart the PC and that fixed it

**Reply by xhofe** [reply]
https://github.com/golang/go/issues/32350#issuecomment-635859542
